src/shared/utils/helpers/get-bool-from-string.ts

```diff
@@ -0,0 +1,9 @@
+export default function getBoolFromString(
+  boolStr?: string,
+): boolean | undefined {
+  return boolStr
+    ? boolStr.toLowerCase() === "true"
+      ? true
+      : false
+    : undefined;
+}
```

Comment:
I think this would be cleaner as:

```suggestion
export default (boolStr?: string) => boolStr?.toLowerCase() === "true";
```

This assumes `undefined` behaves the same as `false` for our purposes.  If being able to return `undefined` is important, you could also do the following that's not too far off from what you have:

```suggestion
export default (boolStr?: string) =>
  boolStr ? boolStr.toLowerCase() === "true" : undefined;
```

I know you're not a big fan of using arrow functions outside of passing functions as arguments, but I think they look pretty clean when the entire function is just evaluating one statement, especially a simple one like this.

Reply:
When I did this, typescript lost all return type information for the function. I'll update it a bit tho.